frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/vm-wrapper.ts

```diff
@@ -221,7 +230,9 @@ export class VMWrapper extends K8sResourceWrapper<VMKind, VMWrapper> implements
       storages.map((storage) => storage.disk),
     );
     this.data.spec.template.spec.volumes = _.compact(storages.map((storage) => storage.volume));
-    this.data.spec.dataVolumeTemplates = _.compact(storages.map((storage) => storage.dataVolume));
+    this.data.spec.dataVolumeTemplates = _.compact(
```

Comment:
we should be consistent with this approach here - pleas update `prependStorage` as well

Reply:
done